docker/docker-entrypoint.sh

```diff
@@ -2,60 +2,60 @@
 
 # Set default values for required env vars
 check=( DB_HOST DB_PORT DB_NAME DB_USER DB_PASSWORD DB_SCHEME WALTZ_FROM_EMAIL WALTZ_BASE_URL CHANGELOG_FILE )
-for x in "${check[@]}"
-do
-  if [ -z "$x" ]; then 
-    echo "$x not provided"
-    case $x in
-      DB_HOST)
-        export DB_HOST="postgres"
-        ;;
-      DB_PORT)
-        export DB_PORT="5432"
-        ;;
-      DB_NAME | DB_USER | DB_PASSWORD | DB_SCHEME)
-        export $x="waltz"
-        ;;
-      WALTZ_FROM_EMAIL)
-        export WALTZ_FROM_EMAIL="help@finos.org"
-        ;;
-      WALTZ_BASE_URL)
-        export WALTZ_BASE_URL="http://127.0.0.1:8080/"
-        ;;
-      CHANGELOG_FILE)
-        export CHANGELOG_FILE="/opt/waltz/liquibase/db.changelog-master.xml"
-        ;;
-    esac
-  fi
+for x in "${check[@]}"; do
+    if [[ -z $(printenv $x) || $(printenv $x) == "" || $(printenv $x) == null ]]; then
```

Comment:
I don't think an environment variable could ever be `null`. It's either not set, or it's a string (which could be empty). So I think you could simplify to:
```suggestion
    if [ -z "$(printenv $x)" ]; then
```